src/api/middlewares/errorHandlingMiddleware.service.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as express from 'express';
+import { Service } from 'typedi';
+
+import { HTTP_CODES } from '../constants';
+
+@Service()
+export class ErrorHandlingMiddlewareService {
+    unhandledErrorsMiddleware(error: Error, req: express.Request, res: express.Response, next: express.NextFunction) {
+        res.status(HTTP_CODES.INTERNAL_SERVER_ERROR).json(error);
```

Comment:
> Add express middlewarewhich will log all unhandled errors and return a standard message with HTTPcode 500(Internal Server Error).

It seems, the message should be "Internal Server Error", but not the error itself

Reply:
KonanMentor fixed